tuned/profiles/unit.py

```diff
@@ -11,8 +11,8 @@ class Unit(object):
 	def __init__(self, name, config):
 		self._name = name
 		self._type = config.pop("type", self._name)
-		self._enabled = config.pop("enabled", True) in [True, "true", 1]
-		self._replace = config.pop("replace", False) in [True, "true", 1]
+		self._enabled = config.pop("enabled", True) in [True, "true", "1"]
+		self._replace = config.pop("replace", False) in [True, "true", "1"]
```

Comment:
Personally, I would append "1" instead of replacing the 1, to be safe. Are you sure there's no way the value will be an integer?

Reply:
I tested this with both python2 and 3 but the value got by this approach was always string, so i think it is safe but to be sure we can just append "1".

Reply:
Well there's also a boolean in the list of possible values. Perhaps there is a reason for each value. Keep in mind that the code is called from multiple places. So I'd prefer to just append the "1".

Reply:
Ok. Solved.